jib-core/src/test/resources/sampleDockerfile

```diff
@@ -4,6 +4,7 @@ COPY libs /app/libs/
 COPY snapshot-libs /app/libs/
 COPY resources /app/resources/
 COPY classes /app/classes/
+COPY exploded-war /jetty/webapps/ROOT/
```

Comment:
I don't think normal jar projects should see `exploded-war` references, and the `/app` locations don't seem right.

Reply:
Normal jar projects won't see `exploded-war`. The `DockerContextGenerator` will add only what it sees. The `sampleDockerfile` is just for testing that the generator has the capacity to handle everything.

Reply:
Also, `/app` will not be there for a WAR project.